### ISSUE TITLE
Improving finding items in large collections

### ISSUE THREAD

**Issue by dur41d** [netlify/netlify-cms#1876]
**Is your feature request related to a problem? Please describe.**

I have a blog like site that has thousands of articles. The articles are organized by categories or sections. The current admin UI only shows a flat list of the article titles. It's not possible to see the sections of each article without opening the article. Also, it's not possible to see the articles of a particular section.

**Describe the solution you'd like**

I would like the ability to filter and group the collection items by their sections so that I can manage large collections more easily.

**Comment by erquhart** [comment]
You can use [`filter`](https://www.netlifycms.org/docs/add-to-your-site/#filter) for that.

**Comment by dur41d** [comment]
Thanks @erquhart. that would work.  The only problem is that you have to re-define with all the fields for each category. But it's not that bad.

In any case, I found that the admin interface is very lacking. For example the articles are sorted by create date in ascending order and it cannot be changed. The admin interface seems to be designed for simple sites with few items and not large collections and relationships.

**Comment by erquhart** [comment]
Yeah, it's not as simple as it seems - sorting the contents of a directory which may or may not be paginated requires a sorting API in the host (GitHub, GitLab, etc) or else downloading of the entire directory of contents up front to sort by. Here's an issue with our latest thoughts: https://github.com/netlify/netlify-cms/issues/54#issuecomment-432471818
